sql_generators/stable_views/__init__.py

```diff
@@ -49,7 +65,10 @@
         distribution_id
     ) AS funnel_derived,
   `moz-fx-data-shared-prod`.udf.distribution_model_installs(distribution_id) AS distribution_model,
-  `moz-fx-data-shared-prod`.udf.partner_org_installs(distribution_id) AS partner_org
+  `moz-fx-data-shared-prod`.udf.partner_org_installs(distribution_id) AS partner_org,
+  mozfun.norm.extract_version(client_info.app_display_version, 'major') as app_version_major,
+  mozfun.norm.extract_version(client_info.app_display_version, 'minor') as app_version_minor,
+  mozfun.norm.extract_version(client_info.app_display_version, 'patch') as app_version_patch,
```

Comment:
`firefox_installer_stable.install_v1` doesn't have `client_info` or a `app_display_version` column so this should probably be `installer_version`